src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
+@import './variables';
+
 body {
   margin: 0;
+  font-family: $calendar-font-family;
+  flex-direction: column;
+  align-items: center;
+  font-size: 30px;
+  display: flex;
 }
+
+.calendar {
```

Comment:
move a calendar to the center of the page
![Screenshot 2024-01-15 at 19 15 50](https://github.com/mate-academy/layout_calendar/assets/117587463/9e32730d-80c5-4255-8023-fbd3b3d30d22)